bittensor/_neuron/text/core_server/run.py

```diff
@@ -160,44 +160,38 @@ def registration_check():
 
         # Check for stake
         def stake_check() -> bool:
-                
             # Check stake.
             uid = metagraph.hotkeys.index(pubkey)
             if metagraph.S[uid].item() < config.neuron.blacklist.stake:
                 raise Exception('Stake blacklist')
             return False
 
-        def validator_check():
-
-            uid = metagraph.hotkeys.index(pubkey)
-            if (metagraph.W[uid] >0).sum() >= n_topk_peer_weights:
-                return False
-
-            raise Exception('Validator blacklist')
-
-
         # Check for time
         def time_check():
             current_time = datetime.now()
-            if pubkey in timecheck.keys():
-                prev_time = timecheck[pubkey]
-                if current_time - prev_time >= timedelta(seconds=config.neuron.blacklist.time):
-                    timecheck[pubkey] = current_time
-                    return False
+            # Only check if the request are forward requests
+            if request_type == 'FORWARD':
+                if pubkey in timecheck.keys():
+                    prev_time = timecheck[pubkey]
+                    if current_time - prev_time >= timedelta(seconds=config.neuron.blacklist.time):
+                        timecheck[pubkey] = current_time
+
+                    else:
+                        timecheck[pubkey] = current_time
+                        raise Exception('blacklist')
```

Comment:
I think last time someone mentioned maybe it would be better if we just return false for these functions instead of raising exception.

Reply:
That was for the synapse checks; we need the function to raise an expectation to reject the requests in the interceptor :)